src/main/kotlin/com/autonomousapps/advice/Advice.kt

```diff
@@ -69,6 +69,11 @@ data class Advice(
       dependency = dependency,
       fromConfiguration = dependency.configurationName, toConfiguration = toConfiguration
     )
+
+    @JvmStatic
+    fun ofCompileOnly(dependency: Dependency) = ofCompileOnly(
```

Comment:
Is this really needed? I guess it's a bit more convenient, but I don't want to bake in the assumption that we only advice `compileOnly` vs `debugCompileOnly`, etc. So I think it's just easier in the long-run to keep using `ofChange()`

Reply:
To be clear, you want me to remove both:
```
   @JvmStatic
    fun ofCompileOnly(dependency: Dependency, toConfiguration: String) = Advice(
      dependency = dependency,
      fromConfiguration = dependency.configurationName, toConfiguration = toConfiguration
    )

    @JvmStatic
    fun ofCompileOnly(dependency: Dependency) = ofCompileOnly(
      dependency = dependency, toConfiguration = "compileOnly"
    )
```
Right ?

Reply:
yes